tf_files/aws/commons/variables.tf

```diff
@@ -133,3 +133,6 @@ locals {
   # kube-aws does not like '-' in cluster name
   cluster_name = "${replace(var.vpc_name, "-", "")}"
 }
+
+variable "squid-nlb-endpointservice-name" {
+  }
```

Comment:
We should have a default value here that points at the CSOC squid service?

Reply:
done.